python/ray/data/_internal/planner/randomize_blocks.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable, List, Optional, Tuple
+
+from ray.data._internal.execution.interfaces import RefBundle
+from ray.data._internal.stats import StatsDict
+
+
+def generate_randomize_blocks_fn(
+    seed: Optional[int],
+) -> Callable[[List[RefBundle]], Tuple[List[RefBundle], StatsDict]]:
+    """Generate function to randomize order of blocks."""
+
+    def fn(refs: List[RefBundle]) -> Tuple[List[RefBundle], StatsDict]:
```

Comment:
This function is refactored from
https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/stage_impl.py#L80 and 
https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/block_list.py#L228